test/fixtures/api/swagger.yaml

```diff
@@ -525,6 +555,54 @@ definitions:
         enum:
           - other
 
+  TrialPublication:
+    required:
+      - attributes
+    properties:
+      attributes:
+        $ref: '#/definitions/Publication'
+
+  Publication:
+    required:
+      - id
+      - source_url
+      - title
+      - abstract
+    properties:
+      id:
+        type: string
+      source:
+        $ref: '#/definitions/Source'
+      created_at:
+        type: string
+        format: date-time
+      updated_at:
+        type: string
+        format: date-time
+      source_url:
+        type: string
+      title:
+        type: string
+      abstract:
+        type: string
+      authors:
+        type: array
+        items: {
+          "type": "string"
+        }
```

Comment:
Same issue here with, instead of `items: { "type": "string" }`, use `items: type: string` (with newlines)